bower.json

```diff
@@ -19,7 +19,7 @@
 		"qunit-assert-close": "JamesMGreene/qunit-assert-close#v1.1.1",
 		"qunit-composite": "JamesMGreene/qunit-composite#v1.1.0",
 		"requirejs": "2.1.14",
-	  	"globalize": "1.0.0",
+	  	"globalize": "globalize#1.1.0-rc.6",
```

Comment:
The post you've reviewed is certainly a big portion of it, but 1.1.0 should include more (e.g., unit formatting). You are welcome to help. :)